draft-huitema-quic-in-space.md

```diff
@@ -78,15 +78,48 @@ network management considerations are out of scope of this document.
-
-Obviously, depending on the location of the asset, the type of orbit, the number of orbiters and assets, etc, that pattern will be more complex and different. However, it will still show an important jump, like from 8 minutes to 1 hour, in the RTT.
-
-The bandwidth of a path in deep space may change based on the conditions. However, experience with Mars (ref to study) shows that it is pretty constant for the same orbiter links to Earth and to the assets. However, if a different orbiter is used on the return path and/or if  optical links are used on some legs where radio links were used previously, then bandwidth would change. However, this is pretty theorical, as paths and routing is very static and planned in advance in space. Therefore, a bandwidth change in the path within a short period of time, or within the time of some packets exchange will be exceptional. If a connection is kept for weeks or months, then it is possible then a few times the path bandwidth will change.
+# Path Changes {#path-changes}
+
+The movement of planets or their satellites implies that the network topology
+will change over time. For example, a station on the surface of Mars will only
+"see" Earth for half a Martian day. or rather sol. If it only relied on direct
+line communication, contact would be lost for half a day. We expect that this
+periodic absence of connectivity will be palliated by communication
+infrastructure such as satellites orbiting Mars. We expect similar satellites
+play a role on the Moon, if we want to enable communication with stations located
+on its hidden face.
+
+We can speculate that at some point in the future, constellations of satellites
+will orbit Mars or the Moon, and provide high quality communication similar to
+what we see on Earth. However, we also expect that the first generation of
+or orbiters will be much less complex. Stations on the hidden side
+of Mars or the Moon will send packets to an orbiting satellite, the
+satellite will store them until relay station
+or maybe Earth is visible, and then forward the stored packets. The packets
+could be stored for fraction of an orbit, probably up to 1 hour assuming
+a 2 hours orbital time for the orbiters.
+
+A station on the surface of Mars could communicate directly with Earth during
```

Comment:
the current rovers do not talk to Earth directly. They may receive commands (at very low bitrate) from Earth, but all comms to Earth goes through an orbiter. The reason is the energy needed and the size of the antenna needed. Even more, it happens sometimes that an overpass of a satellite is not used because the rover does not have enough energy gathered from the Sun during the previous period.   Stations may be different, but...